operations/helm/charts/mimir-distributed/CHANGELOG.md

```diff
@@ -12,7 +12,7 @@ Entries should be ordered as follows:
 Entries should include a reference to the Pull Request that introduced the change.
 
 ## main / unreleased
-
+* [ENHANCEMENT] Check for the containerSecurityContext for privileged value.
```

Comment:
A couple of nits:
1. Please add the PR number at the end of the CHANGELOG entry (e.g. `#2112`)
2. Please keep CHANGELOG entries ordered by type, see: https://github.com/grafana/mimir/tree/main/docs/internal/contributing#changelog